### PR TITLE
(doc) Adds examples of running the service created

### LINKED ISSUE #24: Add explicit ordering to README.md
It would be helpful if the README file had a complete example containing both a `systemd::unit_file` and a `service` resource. In particular, it wasn't immediately obvious that the ordering needs to be made explicit:

```puppet
::systemd::unit_file { 'foo.service':
 source => "puppet:///modules/${module_name}/foo.service",
}

service { foo:
  enable  => true,
  ensure  => running,
  require => Systemd::Unit_file['foo.service']
}
```

Without the explicit ordering, Puppet will attempt to start the service before the unit file has been created.

### PR DESCRIPTION
**Pull request by petems**
* Closes #24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,9 @@ Let this module handle file creation and systemd reloading.
 ::systemd::unit_file { 'foo.service':
  source => "puppet:///modules/${module_name}/foo.service",
 }
+~> service {'foo'
```
Comment (bastelfreak) [inline README.md]: there is a : missing?

File: README.md [hunk 2]
```diff
@@ -33,6 +36,11 @@ file { '/usr/lib/systemd/system/foo.service':
   source => "puppet:///modules/${module_name}/foo.service",
 } ~>
 Exec['systemctl-daemon-reload']
+
+service {'foo'
```
Comment (bastelfreak) [inline README.md]: here as well?

**Comment by bastelfreak** [comment]
Hi @petems, thanks for the patch. Can you take a look at the comments?

By the way, what is in your opinion better?

```puppet
include ::systemd
```

or

```puppet
include "::systemd"
```

**Comment by bastelfreak** [comment]
@petems ping :)

**Comment by raphink** [comment]
@petems could you address @bastelfreak's comments please?

**Comment by petems** [comment]
@raphink @bastelfreak Done! 👍

**Comment by bastelfreak** [comment]
thanks!

**Pull request merged**